packages/core/src/js/integrations/devicecontext.ts

```diff
@@ -83,7 +84,11 @@ async function processEvent(event: Event): Promise<Event> {
     ? native['breadcrumbs'].map(breadcrumbFromObject)
     : undefined;
   if (nativeBreadcrumbs) {
-    event.breadcrumbs = nativeBreadcrumbs;
+    const maxBreadcrumbs = getClient()?.getOptions().maxBreadcrumbs ?? 100; // Default is 100.
```

Comment:
@krystofwoldrich Do you think it's worth caching this value? It is quite unlikely that this option will change after the SDK initializes.

Reply:
I think it's fine without, it's just reading value from object.

But what we should do it read the client from the function args. To avoid the lookup of the global client.

We currently use only the first arg of the function.

https://github.com/getsentry/sentry-javascript/blob/79818ffc304041238b948c20db2aa9a25458e798/packages/types/src/integration.ts#L53 

```js
processEvent?(event: Event, hint: EventHint, client: Client): Event | null | PromiseLike<Event | null>;
```